### ISSUE TITLE
Feature: Bring back storage and network details as sensors, not attributes

### ISSUE THREAD

**Issue by CamFlyerCH** [LAB02-Research/HASS.Agent#203]
Sorry, but for me this last change to move sensors of network and storage to attributes was a big mistake. To have the functionality back I would now need to create an template sensor for at least 18 Sensors ! Without that no data is logged in Grafana anymore.
I installed the release 2022.12.0 and everything is back to normal. I will not install any newer version without a fix to have my sensors back (at least as an option).

**Comment by LAB02-Admin** [comment]
Hey @CamFlyerCH, 'big mistake' might be a bit harsh, but I get your point, and it's something I thought about as well. Thing is that as it was, there were just too many sensors being created. I'll think about the 'publish as individual sensors' option.

I'd rather have HA implement this so the user can pick individual attributes, like this feature request:

https://community.home-assistant.io/t/easy-creation-of-sensors-from-attributes/429646/10

**Comment by mvdwetering** [comment]
It would be possible to create the sensors disabled by default so people that want to use them can enable them.

Or maybe like how the Mikrotik Router custom integration does it. There you can configure if you want sensors. or switches for certain functionality
.
![image](https://user-images.githubusercontent.com/732514/199087065-260072e4-8991-4750-a758-da72d61a71ab.png)

**Comment by LAB02-Admin** [comment]
Agreed. The thing is; I'm a single developer with a ton of pending features/bugfixes and only sparetime to spend on this project. This would mean quite a bit of extra work (more so than creating 18 template sensors I think). So I'll keep it in mind, as it'd be a nice feature, but it could take a while :(
